src/Tasks/ManifestUtil/SecurityUtil.cs

```diff
@@ -705,7 +705,7 @@ public static void SignFile(X509Certificate2 cert, Uri timestampUrl, string path
                         CmiManifestSigner2 signer;
                         if (useSha256 && rsa is RSACryptoServiceProvider rsacsp)
                         {
-                            RSACryptoServiceProvider csp = SignedCmiManifest2.GetFixedRSACryptoServiceProvider(rsacsp, useSha256);
+                            using RSACryptoServiceProvider csp = SignedCmiManifest2.GetFixedRSACryptoServiceProvider(rsacsp, useSha256);
                             signer = new CmiManifestSigner2(csp, cert, useSha256);
```

Comment:
This doesn't look correct. The ownership of `csp` is transferred to `signer` so we should not be disposing of it here.